src/css/components/_progress.scss

```diff
@@ -15,6 +15,7 @@
 .video-js .vjs-progress-control {
   @include flex(auto);
   @include display-flex(center);
+  margin: 0 0.45em 0 0.45em;
```

Comment:
If there isn't any margin added it makes it very difficult to access the surrounding components because there is only 1px between the volume and the seekbar at position 0.

I didn't quite follow the comment.